project_generator_definitions/mcu/st/bluenrg2.yaml

```diff
@@ -0,0 +1,7 @@
+mcu:
+    vendor:
```

Comment:
this is not MCU , not required to be here. you can create our own project with this board and just use the MCU you are adding below ?